src/go/rdctl/pkg/autostart/autostart_windows.go

```diff
@@ -0,0 +1,83 @@
+package autostart
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+const regPath = `C:\Windows\system32\reg.exe`
+
+func EnsureAutostart(autostartDesired bool) error {
```

Comment:
General question for this function: do you have to deal with ucs-2 output from the calls to `reg.exe` ?

Reply:
I've opted to use the registry library that @jandubois suggested. This should take care of any concern about encoding